EKF/control.cpp

```diff
@@ -1093,14 +1083,13 @@ void Ekf::controlHeightFusion()
 	}
 
 	if (isTimedOut(_time_last_hgt_fuse, 2 * RNG_MAX_INTERVAL) && _control_status.flags.rng_hgt
-	    && (!_range_data_ready || !_rng_hgt_valid)) {
+	    && (!(_range_data_ready && _rng_hgt_valid))) {
```

Comment:
Note that I made this change because I planned to change it to a single flag that contains the result of `_range_data_ready && _rng_hgt_valid` that we use everywhere